resources/services/elasticbeanstalk/application_versions.go

```diff
@@ -0,0 +1,149 @@
+			},
+			{
+				Name:        "date_updated",
+				Description: "The last modified date of the application version.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "description",
+				Description: "The description of the application version.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "source_build_information_source_location",
+				Description: "The location of the source code, as a formatted string, depending on the value of SourceRepository  * For CodeCommit, the format is the repository name and commit ID, separated by a forward slash",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("SourceBuildInformation.SourceLocation"),
+			},
+			{
+				Name:        "source_build_information_source_repository",
+				Description: "Location where the repository is stored.  * CodeCommit  * S3  This member is required.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("SourceBuildInformation.SourceRepository"),
+			},
+			{
+				Name:        "source_build_information_source_type",
+				Description: "The type of repository.  * Git  * Zip  This member is required.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("SourceBuildInformation.SourceType"),
+			},
+			{
+				Name:        "source_bundle_s3_bucket",
+				Description: "The Amazon S3 bucket where the data is located.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("SourceBundle.S3Bucket"),
+			},
+			{
+				Name:        "source_bundle_s3_key",
+				Description: "The Amazon S3 key where the data is located.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("SourceBundle.S3Key"),
+			},
+			{
+				Name:        "status",
+				Description: "The processing status of the application version",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "version_label",
+				Description: "A unique identifier for the application version.",
+				Type:        schema.TypeString,
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+
+func fetchElasticbeanstalkApplicationVersions(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	var config elasticbeanstalk.DescribeApplicationVersionsInput
+	c := meta.(*client.Client)
+	svc := c.Services().ElasticBeanstalk
+
+	for {
+		output, err := svc.DescribeApplicationVersions(ctx, &config, func(options *elasticbeanstalk.Options) {
+			options.Region = c.Region
+		})
+		if err != nil {
+			return diag.WrapError(err)
+		}
+
+		for i := range output.ApplicationVersions {
+			if output.ApplicationVersions[i].BuildArn == nil {
```

Comment:
These `nil` values were failing the integration tests.

Please forgive my rusty `go` if I didn't follow a convention here

Reply:
what you would do here is add an IgnoreInTests to the column